src/qililab/analog/annealing_program.py

```diff
@@ -108,9 +110,13 @@ def get_waveforms(self, crosstalk_matrix: CrosstalkMatrix | None = None) -> dict
                 bus_to_flux_map[flux_bus.bus] = flux_bus.flux
         flux_to_bus_map = {v: k for k, v in bus_to_flux_map.items()}
 
+        # add padding to waveforms if duration is not multiple of minimum clock time
+        padded_ns = 0
+        if len(self._transpiled_program) % minimum_clock_time != 0:
+            padded_ns = 4 - len(self._transpiled_program) % 4 if len(self._transpiled_program) % 4 != 0 else 0
```

Comment:
Question: this `4` is a padding that's required by a specific device manufacturer, right? Could another device manufacturer could have a different padding?

If so, maybe it could make sense to put it in a constant (`MANDATORY_PADDING`?), so the code is easier to understand/maintain in the future.

Reply:
yup, well spotted